.github/workflows/dependencies_update.yml

```diff
@@ -65,19 +67,40 @@ jobs:
         run: |
           mvn --batch-mode com.exasol:project-keeper-maven-plugin:fix --projects .
 
-      - name: Generate PR comment
+      - name: Generate Pull Request comment
         id: pr-comment
         # [impl->dsn~dependency-updater.workflow.pull-request-trigger-ci-build~1]
         run: |
           echo 'comment<<EOF' >> "$GITHUB_OUTPUT"
           echo 'This Pull Request was created by [`dependencies_update.yml`](https://github.com/exasol/project-keeper/blob/main/project-keeper/src/main/resources/templates/.github/workflows/dependencies_update.yml) workflow.' >> "$GITHUB_OUTPUT"
-          echo 'It updates dependencies to fix the following vulnerabilities:' >> "$GITHUB_OUTPUT"
-          echo $CREATED_ISSUES | jq --raw-output '. | "* Closes " + .issue_url + " (" + .cve + ")"' >> "$GITHUB_OUTPUT"
+          if [ -n "$CREATED_ISSUES" ]; then
+            echo 'It updates dependencies to fix the following vulnerabilities:' >> "$GITHUB_OUTPUT"
+            echo $CREATED_ISSUES | jq --raw-output '. | "* Closes " + .issue_url + " (" + .cve + ")"' >> "$GITHUB_OUTPUT"
+          else
+            echo 'It updates dependencies.' >> "$GITHUB_OUTPUT"
+          fi
```

Comment:
When taking the PR's title literally -- this is maybe not strictly in the scope.